cluster/terraform_kubernetes/config/prometheus/alertmanager/app_alert.rules.tmpl

```diff
@@ -1,13 +1,20 @@
 groups:
 - name: Container restarts
   rules:
-  - alert: High number of restarted containers
-    expr: sum(kube_pod_container_status_restarts_total) > 1000
+%{ for instance in apps ~}
+  - alert: High number of restarted containers for ${instance.app_name}
+    expr: sum(rate(kube_pod_container_status_restarts_total{container!="",namespace="development",pod=~"apply-review-nnnn-[a-z0-9]+-[a-z0-9]+"  }[5m]) * 300  ) by (namespace,container) > ${instance.max_crash_count}
     for: 5m
+    annotations:
+      summary: High number of restarted containers for ${instance.app_name}
+      description: "The rate of container restart has been above {{ ${instance.max_crash_count} | humanizePercentage }} in the last 5 minutes (current value: {{ $value | humanizePercentage }})"
```

Comment:
max_crash_count is not a percentage. "| humanizePercentage" cannot be used

Reply:
humanizePercentage removed